### PR TITLE
New post!

### PR DESCRIPTION
**Pull request by jjconti**

### CODE REVIEW

File: posts/author-pages-in-nikola-77.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.. title: Author pages in Nikola 7.7
```
Comment (Kwpolska) [inline posts/author-pages-in-nikola-77.rst]: change all the names to be *7.7.0*

File: posts/author-pages-in-nikola-77.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+.. code-block:: mako
+
+            % if author_pages_generated:
+                <a href="${_link('author', post.author())}">${post.author()}</a>
+            % else:
+                ${post.author()}
+            % endif
+
+You can see the feature in action in `Nikola blog <https://getnikola.com/blog/>`_: `author page example <https://getnikola.com/authors/roberto-alsina.html>`_.
```
Comment (Kwpolska) [inline posts/author-pages-in-nikola-77.rst]: link to the all authors list, too

**Comment by Kwpolska** [comment]
About your questions in IRC: Coil does not yet support file uploads (it was supposed to, but I didn’t have time nor will to actually implement it). Nikola could support LaTeX input via pandoc, and does support LaTeX math via mathjax.

**Comment by Aeyoun** [comment]
Schedule this for next week. There have been two posts this week already.

**Comment by Kwpolska** [comment]
Will merge on Wednesday.

**Pull request merged**